Lib/test/test_http_cookies.py

```diff
@@ -121,6 +121,18 @@ def test_set_secure_httponly_attrs(self):
         self.assertEqual(C.output(),
             'Set-Cookie: Customer="WILE_E_COYOTE"; HttpOnly; Secure')
 
+    def test_samesite_strict_attrs(self):
+        C = cookies.SimpleCookie('Customer="WILE_E_COYOTE"')
```

Comment:
We could use `self.subTest()` to make the tests shorter: https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests